src/cmd/crun/main.go

```diff
@@ -26,6 +26,10 @@ var (
 	version = "development" // overwritten by build script -ldflags "-X main.version=..."
 )
 
+func init() {
+	runFlags.StringVarP(region, "region", "r", os.Getenv("AWS_REGION"), "Which cloud region to use, or blank for local Docker")
```

Comment:
why init? I had the impression that Go figures these global inits out by itself?

Reply:
Yeah,  I want to allow both the flag set to modify the same global variable, which will use `StringVarP` that does not return any value, so it cannot be part of the list of global variable assignment block, and go flag package examples usually use `init()` function for flag config, which I am following here.